frontend/components/bridge/EstimatedCompletionTime.tsx

```diff
@@ -0,0 +1,59 @@
+import { Flex, Skeleton, Statistic, Typography } from 'antd';
+import React, { useMemo } from 'react';
+import styled from 'styled-components';
+
+const { Text } = Typography;
+const { Countdown } = Statistic;
+
+const EstimatedTimeRow = styled(Flex)`
+  .ant-statistic .ant-statistic-content .ant-statistic-content-prefix {
+    margin-inline-end: 0 !important;
+  }
+  .ant-statistic .ant-statistic-content .ant-statistic-content-suffix {
+    margin-inline-start: 0 !important;
+  }
+`;
+
+/** time: in minutes */
```

Comment:
Seems it's in seconds? According to the hook and that you multiply by 1000 below. Also, better to use `timeSec` or similar instead of having a comment